frontend/public/style/_overrides.scss

```diff
@@ -213,53 +213,119 @@ tags-input .autocomplete .suggestion-item em {
 
 // PatternFly 4 nav overrides
 
-$pf-l-page__active-sidebar--BackgroundColor: rgba(255,255,255,.07);
+// PatternFly 4 nav overrides
 
-.pf-c-nav__link {
-  --pf-c-nav__list-link--Color: #{$color-pf-black-300};
-}
+@import '../node_modules/patternfly/dist/sass/patternfly/_color-variables';
 
-.pf-c-nav__list {
-  .pf-c-nav__link {
-    &:active {
-      --pf-c-nav__list-link--active--Color: #{$color-pf-white};
-    }
-    &:focus {
-      --pf-c-nav__list-link--focus--Color: #{$color-pf-white};
+.pf-l-page__sidebar {
```

Comment:
Throughout the file:  we aim to alphabetize rules by selector and declarations by property.